README.md

```diff
@@ -1,20 +1,60 @@
-# Tripper
+Tripper
+=======
+_Triplestore wrapper for Python providing a simple and consistent interface to a range of triplestore backends._
 
-**A triplestore wrapper for Python.**
+[![PyPI](https://img.shields.io/pypi/v/tripper?logo=pypi)](https://pypi.org/project/tripper)
+[![Documentation](https://img.shields.io/badge/documentation-informational?logo=github)](https://emmc-asbl.github.io/tripper/latest/)
+![CI tests](https://github.com/EMMC-ASBL/tripper/workflows/CI%20-%20Tests/badge.svg)
 
-_The best ride when handling any triplestore._
```

Comment:
You didn't like this tag-line?

Reply:
No problem. We can keep it as a slogan. 
I just wanted to have a more informative description as well.